### ISSUE TITLE
More microstutter than on the Gear VR S7/S8. Maybe the sys_ticrate has to be changed?

### ISSUE THREAD

**Issue by neitey** [DrBeef/QuakeQuest#8]
The microstutter reminds me of the stutter we had before finding out about the "sys_ticrate" "0.0166667" cvar. Since the game now runs with 72hz instead of 60hz (GearVR), maybe a different sys_ticrate value has to be applied? Will test this out ...

**Comment by neitey** [comment]
The microstutter reminds me of the stutter we had before finding out about the "sys_ticrate" "0.0166667" cvar. Since the game now runs with 72hz instead of 60hz (GearVR), maybe a different sys_ticrate value has to be applied? Will test this out ...

**Comment by DrBeef** [comment]
I did change all instances of sys_ticrate to match the new 72hz, but if you
find anything amiss please let me know

**Comment by neitey** [comment]
I just did a quick spin up of the latest Quake version for the Gear VR and it runs much smoother, butter smooth, whatever phone is used (S7/S8)

I did some testing with QQuake. I gave it a fresh install, not mods, no hd textures. The constant jittering is still there, no change.

I undersampled to 0.7. The stutter is still there, no change at all. 

Seems like the exact same phenomena we encountered on the gearvr. There the stuttering was also not processor performance related and looked and felt the exact same way.

I will try around with the "sys_ticrate" cvar which fixed the issue on the gearVR version.
